src/usr/local/www/widgets/widgets/netgate_services_and_support.widget.php

```diff
@@ -40,9 +40,10 @@
 
 if ($_REQUEST['ajax']) {
 
-	// Retrieve the support data from Netgate.com if the support data file does not exist,
-	// or if it is more than a day old
-	if (!file_exists($supportfile) || ( time()-filemtime($supportfile) > $refreshinterval)) {
+	// Retrieve the support data from Netgate.com if
+	// the support data file does not exist, or
+	// if it is more than a day old and the URL seems resolvable
+	if ( !file_exists($supportfile) || ( time()-filemtime($supportfile) > $refreshinterval && is_url_hostname_resolvable($FQDN) ) ) {
```

Comment:
You need one more set of parenthesis, around `time()-filemtime($copyrightfile) > $refreshinterval`.

Also, this should be broken up into multiple lines. Additional lines should be indented four spaces in below the `if`.